compiler/src/dmd/escape.d

```diff
@@ -1009,9 +1009,9 @@ bool checkNewEscape(ref Scope sc, Expression e, bool gag)
         bool escapingRef(VarDeclaration v, FeatureState fs)
         {
             const(char)* msg = v.isParameter() ?
-                "copying `%s` into allocated memory escapes a reference to parameter `%s`" :
-                "copying `%s` into allocated memory escapes a reference to local variable `%s`";
-            return setUnsafePreview(&sc, fs, gag, e.loc, msg, e, v);
+                "escaping a reference to parameter `%s` by copying `%s` into allocated memory" :
+                "escaping a reference to local variable `%s by copying `%s` into allocated memory";
```

Comment:
Hi @dkorpel - this caused a diagnostic regression - there's a missing closing backtick in this format string.

Are you able to fix this?  Alternatively can include it in the bundle of "broken C++ ABI" things I'm preparing.

Reply:
https://github.com/dlang/dmd/pull/20677